core/src/main/java/org/elasticsearch/search/internal/ShardSearchLocalRequest.java

```diff
@@ -210,6 +219,9 @@ protected void innerReadFrom(StreamInput in) throws IOException {
         if (in.getVersion().onOrAfter(Version.V_5_6_0)) {
             clusterAlias = in.readOptionalString();
         }
+        if (in.getVersion().onOrAfter(Version.V_6_0_0_beta1)) {
+            strict = in.readOptionalBoolean();
+        }
```

Comment:
Cut/paste error. Chatted with @colings86 and we agreed 6.1 - will add a separate note on this PR with more details on this decision.